client/components/domains/domain-suggestion/test/index.js

```diff
@@ -1,3 +1,6 @@
+/** @jest-environment jsdom */
+jest.mock( 'components/plans/premium-popover', () => require( 'components/empty-component' ) );
```

Comment:
I like that this was moved to `components/` – it’s a component after all :) And it’s easier to remember where does it live.